pmm-app/tests/pages/pmmInventoryPage.js

```diff
@@ -65,6 +65,15 @@ module.exports = {
     }
   },
 
+  async getNotRunningAgentsServiceId() {
+    let serviceIDs = await I.grabTextFromAll('//span[contains(text(), \'status: WAITING\')]/preceding-sibling::span[contains(text(), \'service_id:\')]');
```

Comment:
I think you can improve the locator and since you use the same locator at 3 places, this is not a really an efficient way to handle the locator for Agent Status, try to make it more generic and please use `locate` it would make this readable